command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Fri, Jul 29, 2016 at 02:56:17PM -0700, Mrunal Patel wrote:
> > @@ -0,0 +1,189 @@
> > +# Operations
> > +
> > +A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
>
> I think this should be a part of the runtime spec. Without this CLI,
> there will be no compliance testing or interoperability between
> runtimes at all.

I'm ok having this be part of the command-line spec, but the
resistance is that it's hard to find wording that makes everyone
comfortable with container-config-compliant projects that don't
implement the API [1].  I agree that such projects exist, although (in the
absence of another OCI API) they won'y be certifiable or interoperable
for callers [2].

[1]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72829796
[2]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72693334